Uplift/Views/GymDetail/GymDetailTimeInfoView.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  GymDetailTimeInfoView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/22/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class GymDetailTimeInfoView: UILabel {
+
+    private var facility: Facility
+    private var selectedDayIndex = 0
+    private var displayedText = ""
+    private var timesText = NSMutableAttributedString()
+    private var paragraphStyle = NSMutableParagraphStyle()
+
+    // MARK: - Init
+    init(facility: Facility) {
+        self.facility = facility
+        super.init(frame: CGRect())
+
+        timesText.mutableString.setString(displayedText)
+
+        paragraphStyle.lineSpacing = 5.0
+        paragraphStyle.maximumLineHeight = 26
+        paragraphStyle.alignment = .center
+
+        attributedText = timesText
+        backgroundColor = .primaryWhite
+        numberOfLines = 0
+        font = ._16MontserratRegular
+        textColor = .primaryBlack
+        textAlignment = .center
+
+        updateAppearance()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Update
+    private func updateAppearance() {
+        // func call to update tags
+        updateTimes()
+    }
+
+    private func updateTimes() {
+        displayedText = makeDisplayText(dayIndex: selectedDayIndex)
+        updateAttributedText()
+    }
+
+    private func updateTags() {
+        let tagLabelWidth = 81
+        let tagSideOffset = 25.0
+        let textLineHeight = font.lineHeight
+
+        let info = facility.miscInformation
```

Comment:
heads up that this is going to change with the new backend models -- facility no longer has `miscInformation` directly as a property

Reply:
I was thinking of fixing that in another PR, just to get this merged in first